Text/Hamlet.hs

```diff
@@ -263,8 +267,13 @@ docToExp env hr v (DocContent c) = contentToExp env hr v c
 
 contentToExp :: Env -> HamletRules -> Scope -> Content -> Q Exp
 contentToExp _ hr _ (ContentRaw s) = do
+#if MIN_VERSION_bytestring(0,11,2)
+    os <- [|unsafeByteString|]
+    s' <- lift $ TE.encodeUtf8 (pack s)
+#else
     os <- [|preEscapedText . pack|]
     let s' = LitE $ StringL s
```

Comment:
Looking at the [warnings on `unsafeByteString`](https://www.stackage.org/haddock/lts-24.14/blaze-markup-0.8.3.0/Text-Blaze.html#v:unsafeByteString), this seems reasonable - we have the same issue with illegal HTML characters as `preEscapedText` and we don't need to worry about encoding (it's a string literal)